docs/markdown/android/iconbutton.md

```diff
@@ -0,0 +1,114 @@
+---
+title: IconButton
+description: IconButton allows users to take actions and make choices with a single click or tap. IconButtons use icons instead of text to convey available actions on a screen. IconButton is typically found in forms, dialogs and toolbars.Some buttons are specialized for particular tasks, such as navigation or presenting menus.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/04/af/d1/04afd13f2e6d2e3a659f133cf1c8b360.png" width={48} height={48} alt="IconButton component"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
```

Comment:
These headings should change too, "When to use" and "When not to use"

Reply:
Nice catch! Fixed!